lib/es/importDataFromFile.js

```diff
@@ -40,20 +41,35 @@ async function getPreviousIndex(alias) {
   return indexCurrent || alias;
 }
 
-async function importDataFromFile(filename, index, threshold) {
+async function importDataFromFile(filename, indexAlias, threshold) {
   try {
-    const latestIndex = `${index}_${getDateStamp()}`;
+    // if running locally, wait for ES to start
+    if (config.env === 'development') {
```

Comment:
Does this conditional need to be here? It should be ok just running
```js
await esClient.waitForEsToStart();
```

Reply:
The ES health status is being used to determine if Elasticsearch is running.
A cluster will return 'red' when some of the nodes are unhealthy even though the cluster is usable and performant. This is what we are currently seeing on the dev environment as some nodes are shut down.

The intent is to ensure the local ES has started when running the compose file locally as it can take longer to start ES than to download the source files.

We can find another method of checking if the Elasticsearch server is working if there is value in having a check against the live service.

Reply:
Ok, I didn't realise the cluster was returning a status of red. Perhaps that is something we should look into...
Seems sensible to keep this as it is for the time being.